nativelib/src/main/scala/scala/scalanative/native/stdlib.scala

```diff
@@ -2,8 +2,8 @@ package scala.scalanative
 package native
 
 @extern object stdlib {
-  var __stderrp: Ptr[_] = extern
-  var __stdoutp: Ptr[_] = extern
+  var stderr: Ptr[_] = extern
+  var stdout: Ptr[_] = extern
```

Comment:
This is in fact __stdoutp on Darwin. We really need to get some form of conditional compilation (https://github.com/scala-native/scala-native/issues/58) working sooner rather than later.